Cargo.toml

```diff
@@ -15,21 +15,21 @@ edition = "2018"
 [dependencies]
 indexmap = "1"
 grammer = "0.0.1"
-proc-macro2 = "0.4.30"
-proc-quote = "0.2.2"
+proc-macro2 = "1.0.6"
+proc-quote = "0.3.0"
```

Comment:
These versions are not compatible, you can see it in the Travis CI build log:
```
   Compiling proc-macro2 v0.4.30
   Compiling proc-macro2 v1.0.6
```

Reply:
I don't know why -- proc-quote 0.3 depends on proc-macro2 ^1.0

Reply:
If you try to build this locally, `Cargo.lock` should tell you where the two versions come from.

Reply:
Also, [`cargo tree`](https://crates.io/crates/cargo-tree)` -p proc-macro2 -i` should show exactly where it comes from.